server/middleware/workstack.js

```diff
@@ -52,11 +52,21 @@ async function getTeamMembers(req, res, next) {
     }
 }
 
+async function getMoveTeamOptions(req, res, next) {
+    try {
+        const response = await req.listService.fetch('MOVE_TEAM_OPTIONS', req.params);
+        res.locals.workstack.moveTeamOptions = response;
+        next();
+    } catch (error) {
+        next(error);
+    }
+}
+
 function workstackApiResponseMiddleware(req, res) {
     res.json(res.locals.workstack);
 }
 
-const allocateUser = async (req, res, [endpoint, body, headers]) => {
+const putToCaseworkService = async (req, res, [endpoint, body, headers]) => {
```

Comment:
This isn't strictly descriptive of the behaviour described in the method. If the functionality fails the try and is caught, it will log an error about allocation failure, where the method name suggests it's responsible for more agnostic 'putting'

Reply:
I could make the error message more generic, this would make the error less informative, but it would still have the endpoint in the message. Or do you think it would be better to put the name back and create a dedicated method for move team?

Reply:
I think it might be better to split them out and have the method name more descriptive of the behaviour it is performing. If it was like the previous allocateUser and another for move team, it gives us more flexibility to do more stuff in future for those actions separately.

The other option could be to pass in the error and notification messages as another parameter into the method, but this could make the method more difficult to understand its purpose.

Reply:
I have split these